community/inventory_service/item-schema-automated.json

```diff
@@ -655,8 +691,11 @@
 			"itemdefid": 28,
 			"type": "item",
 			"name_english": "Priority Number One",
```

Comment:
by the way, "Number One" is slang for peeing, just in case there's some equivalent pun that can be made in Japanese

Reply:
oh, I didn't know that. I will change this if I have an idea.